topics/variant-analysis/tutorials/dunovo/tutorial.md

```diff
@@ -104,23 +103,41 @@ The starting point of the analysis is sequencing reads (in [FASTQ](https://en.wi
 
 ## Getting data in and assessing quality
 
-We uploaded the [Schmitt *et al.* 2015](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4414912/) data directly from SRA as shown in [this screencast](https://vimeo.com/121187220).
+You can obtain the data from [Schmitt *et al.* 2015](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4414912/) from Zenodo, or just import the [Galaxy history](https://usegalaxy.org/u/nstoler/h/srr1799908---schmitt-2015) that accompanies this tutorial.
 
-> ### {% icon hands_on %} Hands-on: Importing the raw data
+> ### {% icon hands_on %} Hands-on: Importing the input FASTQ files
 >
-> You can obtain the reads from this dataset by copying [this history](https://usegalaxy.org/u/nstoler/h/srr1799908---schmitt-2015).
+> Option 1: Import the files from Zenodo (this will work with any Galaxy instance):
+> 1. In the Galaxy interface, click on the upload icon.  
+>    \- The one with an up arrow, at the top of the **Tools** panel on the left.
+> 2. In the window that pops up, click the button on the bottom labeled *Paste/Fetch data*.
+> 3. Then paste this link into the text box that appears:  
+>    `https://zenodo.org/record/3554549/files/SRR1799908_forward.fastq?download=1`
+> 4. Name the dataset `SRR1799908_forward`.
+> 5. Repeat steps 2-4 but with the following link, and the name `SRR1799908_reverse`:  
+>    `https://zenodo.org/record/3554549/files/SRR1799908_reverse.fastq?download=1`
+> 6. Then click the *Start* button, then *Close*.
+>
+> Option 2: Obtain the files by importing [this Galaxy history](https://usegalaxy.org/u/nstoler/h/srr1799908---schmitt-2015) (this only works on [usegalaxy.org](https://usegalaxy.org)):
```

Comment:
Would you remind removing this? It unfortunately works only on one of the servers, and we try really hard to make all of the tutorials work on every server.

You've mentioned `usegalaxy.org` in a couple other places too, is it possible you could search for and replace those references with more generic instructions that will work for everyone? E.g. If you copy the workflows you've created into the training material then we can additionally test them with [automated workflow testing](https://github.com/usegalaxy-eu/workflow-testing/) that can let trainers know they work on specific servers.